Snabble/UI/Utilities/UIView+RestrictDynamicTypeSize.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UIView+RestrictDynamicTypeSize.swift
+//  Snabble
+//
+//  Created by Anastasia Mishur on 26.05.22.
+//
+
+import UIKit
+
+extension UIView {
+    func restrictDynamicTypeSize(from min: UIContentSizeCategory?, to max: UIContentSizeCategory?) {
+        self.minimumContentSizeCategory = min
+        self.maximumContentSizeCategory = max
+    }
```

Comment:
Shouldn't this method be marked with an available flag? `maximumContentSizeCategory` and `minimumContentSizeCategory` are iOS 15+ 🤔 is that missing in the public documentation? Did you check it on iOS 14?

https://developer.apple.com/documentation/uikit/uiview/3750924-minimumcontentsizecategory#

Reply:
Fixed for iOS lower then 15+